.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
 /build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
-composer.lock
+/Resources/doc/_build
 /vendor
+composer.lock
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore